zubhub_frontend/zubhub/src/store/actions/projectActions.js

```diff
@@ -208,6 +208,33 @@ export const getProjects = args => {
   };
 };
 
+export const getMoreProjects = args => {
+  return () => {
+    return API.getMoreProjects(args)
+      .then(res => {
+        if (Array.isArray(res)) {
+          return {
+            results: res,
+            loading: false,
+          };
+        } else {
+          res = Object.keys(res)
+            .map(key => res[key])
+            .join('\n');
+          throw new Error(res);
+        }
+      })
+      .catch(error => {
+        if (error.message.startsWith('Unexpected')) {
```

Comment:
The string check here can go wrong if translated, maybe focusing on the error message code can be better

Reply:
okay! Roger that.